### DISCUSSION TITLE
[Improvement] Angular 17 - Component - Use new 'styleUrl' property instead of 'styleUrls'

### DISCUSSION THREAD

**Discussion by xXNickznXx** [cyrilletuzi/vscode-angular-schematics#914] [Ideas]
Angular 17 introduced the new 'styleUrl' property for components, which is also the default when generating with the command.
So it should also be the default in the extension, instead of this:
```javascript
@Component({
  ...
  styleUrls: ['./[name].component.scss'],
  ...
})
```
It should be this:
```javascript
@Component({
  ...
  styleUrl: './[name].component.scss',
  ...
})
```

**Comment by cyrilletuzi** [comment]
Hello,

It is already planned on the roadmap.

Even if it does not apply to this request, one element of clarification: this extension is independent from the Angular CLI. The fact that the Angular CLI does something does not mean that this extension should or will do the same.

Also, this is not a bug, so moving this to Discussions.

**Comment by cyrilletuzi** [comment]
This is available in version 6.1.0.
